src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -953,13 +979,15 @@ public boolean requiresWorkspaceForPolling() {
     private static class CheckOutTask extends UpdateTask implements FileCallable<List<External>> {
         private final UpdateTask task;
 
-         public CheckOutTask(Run<?, ?> build, SubversionSCM parent, ModuleLocation location, Date timestamp, TaskListener listener, EnvVars env) {
+        public CheckOutTask(Run<?, ?> build, SubversionSCM parent, ModuleLocation location, Date timestamp,
```

Comment:
class is private, so it's fine

Reply:
Hmm... So, ignore the previous suggestion, private + getter?